### ISSUE TITLE
Deprecated funktion pix_url

### ISSUE THREAD

**Issue by Nullmann** [moodleuulm/moodle-theme_boost_campus#12]
After installing the plugin and having debug mode enabled, I was greeted by this error message:
```
pix_url is deprecated. Use image_url for images and pix_icon for icons.
```
This function is called 3 times in classes\output\core_renderer.php

Opening a course page, there are broken image-links in the top-left corner:
![screenshot](https://user-images.githubusercontent.com/15816473/34678575-d18dc24e-f493-11e7-857b-ed6d838dacbc.jpg)

Installed moodle version is 3.3, OS is windows (windows installation package)

**Comment by Kathrin84** [comment]
Hi Nullmann,
thanks for your issue. I'm in the middle of preparing the new versions for Moodle 3.3 and 3.4.
The deprecated pix url notice will be handled there.
Regarding the broken images, please let me know if this is still an issue after you installed the soon to come corresponding version of the theme.
Best, Kathrin

**Comment by abias** [comment]
Hi Nullmann, 

quick update: The 3.3 version is now released and the 3.4 version is currently prepared. I would be grateful if you could re-test the 3.3 version to see if the problems which you have reported in this issue are already done or not.

Thanks,
Alex

**Comment by Nullmann** [comment]
With the newest version, the error is not present anymore.
